qemu/tests/thin_provisioning_guest_show_unmap.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from autotest.client.shared import error
+from autotest.client import os_dep
+from avocado.core import exceptions
+from virttest import env_process
+from qemu.tests import thin_provisioning
+
+
+@error.context_aware
+def run(test, params, env):
+    """
+      'thin-provisioning' functions of fixed bugs
+      1)modeprobe scsi_debug
+      2)using lsscsi to check that scsi_debug disk is fine
+      3)check provisioning_mode of the disk in host
+      4)boot up guest
+      5)check provisioning_mode in guest
+      :param test: QEMU test object
+      :param params: Dictionary with the test parameters
+      :param env: Dictionary with test environment.
+      """
+
+    # Destroy all vms to avoid emulated disk marked drity before start test
+    for vm in env.get_all_vms():
+        if vm:
+            vm.destroy()
+            env.unregister_vm(vm.name)
+
+    os_dep.command("lsscsi")
```

Comment:
hi, it is analysied in the thin_provisioning.py , function: get_scsi_disk(session=None)